install_config/redeploying_certificates.adoc

```diff
@@ -533,14 +533,14 @@ $ REGISTRY_HOSTNAME=`oc get route/docker-registry -o jsonpath='{.spec.host}'`
 $ oc adm ca create-server-cert \
     --signer-cert=/etc/origin/master/ca.crt \
     --signer-key=/etc/origin/master/ca.key \
-    --hostnames=$REGISTRY_IP,docker-registry.default.svc.cluster.local,$REGISTRY_HOSTNAME \
+    --hostnames=$REGISTRY_IP,docker-registry.default.svc,docker-registry.default.svc.cluster.local,$REGISTRY_HOSTNAME
     --cert=/etc/origin/master/registry.crt \
     --key=/etc/origin/master/registry.key \
     --signer-serial=/etc/origin/master/ca.serial.txt
 ----
 +
 Run `oc adm` commands only from the first master listed in the Ansible host inventory file,
-by default *_/etc/ansible/hosts_*.
+by default *_/etc/ansible/hosts_*. See the link:https://github.com/openshift/openshift-ansible/blob/release-3.7/playbooks/common/openshift-cluster/redeploy-certificates/registry.yml#L70[Ansible playbook] when redeploying registry certificates. If `docker-registry.default.svc` is not added, the new generated certificate does not have `docker-registry.default.svc` DNS in the SAN.
 
```

Comment:
Do we need the link here? It is a repeat of your code example and doesn't add much (for me, anyway).

I am not sure what this mean: "`docker-registry.default.svc` DNS in the SAN" I see it in the BZ, so maybe it is OK.

Reply:
We don't need such explanation in an official doc IMO